spec/forms/employers_contract_claim_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe EmployersContractClaim, type: :model do
+
+  context 'when correctly populated' do
+
+    it 'with make_employer_contract_claim only' do
+      employers_contract_claim = described_class.new(make_employer_contract_claim: true)
+
+      expect(employers_contract_claim.make_employer_contract_claim).to be true
+    end
+
+    # Unit tests for upload functionality
+    it 'with defend_claim_facts only' do
```

Comment:
Should you have removed this ?  its just empty.

Which makes me wonder if rubocop is setup correctly as I think it should detect this

Reply:
Removed in e8cef58.

Not sure on rubocop, it is definitely working on the repo, perhaps a misconfigured rule?